### ISSUE TITLE
Premature closure of the ductus arteriosus

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7181]
**Preferred term label:**
Premature closure of the ductus arteriosus

**Synonyms**

**Definition (free text, please give PubMed ID)**
Closure of the DA prior to delivery.

see also https://pubmed.ncbi.nlm.nih.gov/152705/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added.
